test/e2e/framework.go

```diff
@@ -199,7 +199,7 @@ func NewFramework(inClusterTestSuite bool) *Framework {
 		}
 
 		var err error
-		contourBin, err = gexec.Build("github.com/projectcontour/contour/cmd/contour")
+		contourBin, err = gexec.Build("github.com/projectcontour/contour/cmd/contour", "-race")
```

Comment:
we can omit this change for this PR, I have a follow up that will add race detection